opt/scopeserver/dataserver/modules/gserver/s_pb2.py

```diff
@@ -1,6949 +1,4367 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: s.proto
-# pylint: skip-file
-# type: ignore
```

Comment:
These should stay to make the linting happier. We should alter the bash script generating these to add them back in and then black the outputs

Reply:
Merged master into this branch ;)

Reply:
Weird, still had to still correct some files...